src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+
+import { Dispatch, SetStateAction } from 'react';
+import { Todo } from '../../types/Todo';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  todos: Todo[];
+  activeTodo: number;
+  filteredField: FilterType;
+  setFilteredField: Dispatch<SetStateAction<FilterType>>;
+  onClearCompleted: () => Promise<void>;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  activeTodo,
+  filteredField,
+  setFilteredField,
+  onClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodo} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {Object.values(FilterType).map(filter => (
+          <a
+            key={filter}
+            href={`#/${filter === FilterType.All ? '' : filter.toLowerCase()}`}
```

Comment:
u can do it easier with `&&` operator

```suggestion
            href={`#/${filter !== FilterType.All && filter.toLowerCase()}`}
```